lbry/wallet/server/db/elasticsearch/search.py

```diff
@@ -236,37 +237,51 @@ async def search_ahead(self, **kwargs):
         limit = kwargs.pop('limit', 10)
         offset = kwargs.pop('offset', 0)
         kwargs['limit'] = 1000
-        query = expand_query(**kwargs)
-        result = (await self.search_client.search(
+        cache_item = ResultCacheItem.from_cache(f"ahead{per_channel_per_page}{kwargs}", self.search_cache)
+        if cache_item.result is not None:
+            to_inflate = cache_item.result
```

Comment:
can `to_inflate` be renamed something else? as to not confuse with the output protobuf inflation, ie what's done with the final result of this call by the client